otaclient/app/proto/_otaclient_v2_pb2_wrapper.py

```diff
@@ -39,6 +43,87 @@
 )
 
 
+# protocols
+
+
+class ECU(_Protocol):
+    ecu_id: str
+
+
+ECUType = _TypeVar("ECUType", bound=ECU)
+
+
+class ECUList(_Protocol[ECUType]):
+    """A type of message that contains a list of ECUType."""
+
+    ecu: _List[ECUType]
+
+    def add_ecu(self, ecu: ECUType):
+        self.ecu.append(ecu)
+
+    def if_contains_ecu(self, ecu_id: str) -> bool:
+        for ecu in self.ecu:
+            if ecu.ecu_id == ecu_id:
+                return True
+        return False
+
+    def find_ecu(self, ecu_id: str) -> _Optional[ECUType]:
+        for ecu in self.ecu:
+            if ecu.ecu_id == ecu_id:
+                return ecu
+
```

Comment:
Add return None when loop is break.
```suggestion
                return ecu
        return None
```

Reply:
Updated in [df9c283](https://github.com/tier4/ota-client/pull/212/commits/df9c283a0f42a2f451135f27f1bfde5b8a8489e7).